src/components/integrated/OakQuizCheckBox/OakQuizCheckBox.tsx

```diff
@@ -0,0 +1,224 @@
+    <OakFlex
+      $flexDirection="column"
+      $minWidth={"all-spacing-20"}
+      $gap={"space-between-s"}
+    >
+      <OakBox>{image}</OakBox>
+      {value}
+    </OakFlex>
+  );
+
+  const inputCheckbox = (
+    <StyledFlexBox
+      $pa="inner-padding-l"
+      $borderRadius={"border-radius-m2"}
+      $borderColor={"border-primary"}
+      $background={disabled ? "bg-neutral-stronger" : "bg-primary"}
+      $flexGrow={1}
+    >
+      <StyledOverlay
+        className="grey-shadow"
+        $position={"absolute"}
+        $left={"space-between-none"}
+        $top={"space-between-none"}
+        $borderRadius={"border-radius-m2"}
+        $width={"100%"}
+        $height={"100%"}
+      />
+
+      <StyledOverlay
+        className="yellow-shadow"
+        $position={"absolute"}
+        $borderRadius={"border-radius-m2"}
+        $left={"space-between-none"}
+        $top={"space-between-none"}
+        $width={"100%"}
+        $height={"100%"}
+      />
+
+      <InternalCheckBoxLabelHoverDecor
+        htmlFor={id}
+        labelGap={"space-between-s"}
+        labelAlignItems={"center"}
+        $color={disabled ? "text-disabled" : "text-primary"}
+        $font={"body-1"}
+        disabled={disabled}
+      >
+        <InternalCheckBoxWrapper
+          size={"all-spacing-7"}
+          iconPadding="inner-padding-ssx"
+          checkedIcon={
+            <OakBox
+              $width="100%"
+              $height="100%"
+              $background={disabled ? "text-disabled" : "text-primary"}
+            >
+              <OakBox
+                $ba={"border-solid-m"}
+                $borderColor={"white"}
+                $width={"100%"}
+                $height={"100%"}
+              />
+            </OakBox>
+          }
+          internalCheckbox={
+            <StyledInternalCheckBox
+              id={id}
+              value={value}
+              disabled={disabled}
+              {...rest}
+              $width={"all-spacing-7"}
+              $height={"all-spacing-7"}
+              $ba={"border-solid-m"}
+              $borderColor="border-neutral"
+              $borderRadius={"border-radius-s"}
+              $checkedBackground={null}
+            />
+          }
+        />
+        {image ? imageContainer : value}
+      </InternalCheckBoxLabelHoverDecor>
+    </StyledFlexBox>
+  );
+
+  const feedbackCheckbox = (
+    <OakFlex
+      $pa="inner-padding-l"
+      $background={"bg-primary"}
+      $flexGrow={1}
+      $gap={"space-between-s"}
+      $borderRadius={"border-radius-m2"}
+      $alignItems={"center"}
+      $font={"body-1"}
+    >
+      <OakBox
+        $position="absolute"
+        $top="all-spacing-0"
+        $left="all-spacing-0"
+        $width="100%"
+        $height="100%"
+        $ba="border-solid-l"
+        $borderRadius={"border-radius-m2"}
+        $borderColor={isCorrect ? "border-success" : "border-error"}
+      />
+      <OakIcon
+        iconName={isCorrect ? "tick" : "cross"}
```

Comment:
@kimon-satan & @messmike My initial instinct here is that the alt text of these icons should be their _meaning_, not their appearance - i.e. `correct` and `incorrect`) respectively. What do you both think?

Reply:
Yeah good shout @rightsaidjames

Reply:
Doh just noticed this